tableformatter.py

```diff
@@ -696,7 +708,7 @@ def Column(col_name: str,
         return col_name, opts
 
 
-def Row(*args, text_color: TableColors=None):
+def Row(*args, text_color: Union[TableColors, str]=None):
```

Comment:
You might want to elaborate in the docstring what kind of *str* might get passed in for ``text_color`` - i.e. it can take ANSI escape codes, but isn't looking for something like 'green'.